src/mjolnir/adminbuilder.cc

```diff
@@ -142,8 +142,14 @@ void buffer_polygon(const polygon_t& polygon, multipolygon_t& multipolygon) {
   auto* outer_ring = geos_helper_t::from_striped_container(polygon.outer());
   std::vector<GEOSGeometry*> inner_rings;
   inner_rings.reserve(polygon.inners().size());
-  for (const auto& inner : polygon.inners())
-    inner_rings.push_back(geos_helper_t::from_striped_container(inner));
+  // there is some weird AppleClang bug where it would iterate over non-existing elements
+  // in the inners() vector and eventually segfault in from_striped_container()
+  std::cout << "Size of polygon.inners(): " << polygon.inners().size() << std::endl;
```

Comment:
I'm not kidding: this line printing to `stdout` is actually making the tests pass! Compiler bug wasting many hours of my life..